boot/modeenv.go

```diff
@@ -376,6 +376,27 @@ func (m *Modeenv) TryModelForSealing() secboot.ModelForSealing {
 	}
 }
 
+func (m *Modeenv) setModel(model *asserts.Model) {
```

Comment:
Our new checker complains that this function is not used yet, do we care?

Reply:
No, that commit was cherry picked from #10469, I didn't want to split further in two to avoid conflicts.